ikasaneip/rest/rest-dashboard/src/main/java/org/ikasan/rest/dashboard/ReplayApplication.java

```diff
@@ -76,17 +86,18 @@ public ResponseEntity harvestReplayEvents(@RequestBody String replayEventsJsonPa
     {
         try
         {
-            List<ReplayEventImpl> replayEvents = this.mapper.readValue(replayEventsJsonPayload
-                , mapper.getTypeFactory().constructCollectionType(List.class, ReplayEventImpl.class));
+            logger.debug(replayEventsJsonPayload);
 
-            logger.info(replayEventsJsonPayload);
+            List<ReplayEvent> replayEvents = this.mapper.readValue(replayEventsJsonPayload
+                , mapper.getTypeFactory().constructCollectionType(List.class, ReplayEventImpl.class));
 
+            this.batchInsert.insert(replayEvents);
         }
-        catch (IOException e)
+        catch (Exception e)
         {
             e.printStackTrace();
             throw new ResponseStatusException(
-                HttpStatus.BAD_REQUEST, "Cannot parse replay JSON!", e);
+                HttpStatus.BAD_REQUEST, "An error has occurred attempting to perform a batch insert of ReplayEvents!", e);
```

Comment:
Given we expect Application/json as a request shouldnt we respond with JSON as well. Providing just text as an answer seems inconsistent. 

Check https://github.com/ikasanEIP/ikasan/blob/2.2.0/ikasaneip/rest/rest-module/src/main/java/org/ikasan/rest/module/FilterApplication.java

```
  return new ResponseEntity(new ErrorDto("fromDateTime or untilDateTime has invalid dateTime format not following yyyy-MM-dd'T'HH:mm:ss."),
                HttpStatus.BAD_REQUEST);
```

Reply:
Taken you direction on this @andrzej-majewski

Reply:
@andrzej-majewski appears that PUT services will not allow you to return json... I have reverted. Please approve so that I can crack on.